cmd/repo_set_default.go

```diff
@@ -34,7 +34,7 @@ The default repository is used when you run the "appsody init" command without s
 				return errors.New("you must specify desired default repository")
```

Comment:
Whilst we're in here this error message doesn't make sense, how about "You must specify the repository to set as the default."